_posts/2021-06-08-whats-coming-in-eslint-8.0.0.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: post
+title: What's coming in ESLint v8.0.0
+tags:
+  - major
+  - breaking changes
+---
+# What's coming in ESLint v8.0.0
+
+Last week, the Technical Steering Committee (TSC) met to finalize the features for ESLint v8.0.0. This feature set had not yet been locked down even as v8.0.0 work began and so this represented an important milestone that allows us to focus on getting this next major release shipped. You can see everything that is planned for v8.0.0 on our [project board](https://github.com/eslint/eslint/projects/8).
+
+We want to share with you the significant changes that will be coming in v8.0.0 so you can better prepare for the changes and understand why the changes are necessary.
+
+## Significant changes in v8.0.0
+
+The following changes are important to understand as we move towards v8.0.0.
+
+### Dropping support for Node.js v10.x and 13.x
+
+Node.js v10.x end-of-life was April 30, 2021 and Node.js v13.x end-of-life was June 1, 2021. As a result, ESLint v8.0.0 [removes support](https://github.com/eslint/eslint/issues/14023) for both of these Node.js versions.
```

Comment:
```suggestion
Node.js v10.x end-of-life was April 30, 2021 and Node.js v13.x end-of-life was June 1, 2020. As a result, ESLint v8.0.0 [removes support](https://github.com/eslint/eslint/issues/14023) for both of these Node.js versions.
```

Also, per https://github.com/eslint/eslint/pull/14592 we will remove support for v15.x too?

Reply:
Good point, I’ll add that.